packages/rancher-istio/charts/Chart.yaml

```diff
@@ -13,7 +13,7 @@ annotations:
   catalog.cattle.io/release-name: rancher-istio
   catalog.cattle.io/ui-component: istio
   catalog.cattle.io/provides-gvr: networking.istio.io.virtualservice/v1beta1
-  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.29.000-rc00
+  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.29.001-rc00
```

Comment:
This is less for review and more for understanding: what does this annotation do?

Reply:
rancher-istio has a dependency on rancher-kiali-server, which has a CRD chart. Since this means that rancher-istio needs to install rancher-kiali-server-crd automatically on the UI in order to have access, we need to add this annotation.

This type of dependency on another package's CRD chart is not very neatly handled by the charts-build-scripts today, which is why we need to manually add this annotation.

Added an issue at https://github.com/rancher/charts-build-scripts/issues/28 to address this.

Reply:
Gotcha - thanks for the echo issue. Both the manual approach and the proposed one in the issue make sense.